addon/utils/controls.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
necessary?

Reply:
No this is not necessary, I will remove it.